ophyd/commands.py

```diff
@@ -538,9 +572,12 @@ def logbook_add_objects(objects, extra_pvs=None):
     msg += '{:-^120}\n'.format('')
 
     # Make a list of all PVs and positioners
-    pvs = [o.report['pv'] for o in objects]
+    reports = [o.report for o in objects]
+    pvs = [report['pv'] for report in reports]
     names = [o.name for o in objects]
-    values = [str(o.value) for o in objects]
+    values = [str(v) for report in reports
+              for k, v in report.items() if k != 'pv']
```

Comment:
@klauer : @dchabot wanted to have a bit more of a think about this line as it is pretty awful.  `o.value` was raising if a motor was disconnected, so not sure that this is the right place to handle that.  If @dchabot signs off on this, then this PR  looks good to me